### ISSUE TITLE
Change the Discount code from 51C to 51G for kanes

### ISSUE THREAD

**Issue by kbower** [kbower/tickettest1#903]
_Imported from trac ticket 903.
Created by **sshah**
Opened in trac: May 23, 2011, 4:22:01 PM
Last modified in trac: May 24, 2011, 8:28:25 AM_
***
Kent,

Good afternoon, can you change the 51C discount code to 51G as this has already been communicated to the company as may also be printed on promotional materials?

Thanks,

Wayne

**Comment by kbower** [comment]
_**sshah** trac comment on May 23, 2011, 4:23:37 PM:_

(Changeset [[1512]](../commit/f4b1337a7dbcbcc7c0cad2431b4f884202271a92))
Fixes #903 change the 51$ discount code from 51C to 51G

**Comment by kbower** [comment]
_**sshah** trac status change on May 23, 2011, 4:23:37 PM:_ **closed**

**Comment by kbower** [comment]
_**kbower** trac comment on May 24, 2011, 8:25:49 AM:_

Wayne, they are all there, but 51G had already been added (possibly through GERS?) and in fact applied to one order.  It was not set up properly though, so I deleted it so that it would be readded properly.

You will therefore need to reset its expiry date.

Note that the order to which it had been applied will have it as $51 off a line item instead of off the order.  This was order 0523111YO75.

Kent
